notebooks/official/workbench/sentiment_analysis/Sentiment_Analysis.ipynb

```diff
@@ -1,16 +1,68 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    model.wait()
</pre><p>This line is unneeded because the method is sync by default.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>